bootstrap_cfn/utils.py

```diff
@@ -15,3 +15,19 @@ def wrapper(*args, **kwargs):
                 time.sleep(interval)
         return wrapper
     return decorate
+
+def connect_to_aws(module, instance):
+    try:
+        conn = module.connect_to_region(
+            region_name=instance.aws_region_name,
+            profile_name=instance.aws_profile_name
+        )
+        return conn
+    except NoAuthHandlerFound:
```

Comment:
Rather than printing an error can you add an error class to errors.py and raise that with a message